dockstore-common/src/test/java/io/dockstore/common/BasicPostgreSQL.java

```diff
@@ -130,11 +130,10 @@ public void clearDatabase() {
         }
     }
 
-    protected boolean runUpdateStatement(String query, Object... params) {
+    protected int runUpdateStatement(String query, Object... params) {
```

Comment:
I assume this returns the number of rows that were updated? Might be nice to have a comment to that effect, although I guess most people will just know that.